### PR TITLE
Add CSS grouping selectors compat data

### PR DESCRIPTION
**Pull request by ramiy**
MDN:
https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/Grouping_selectors

W3C Specifications:
https://www.w3.org/TR/CSS1/#grouping

A checklist to help your pull request get merged faster:
- [ ] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Hey @ramiy, thanks for your PR!  It looks like you're using the wrong URL for the MDN docs.  Please review the output from `npm test` and fix any errors that come up!

File: css/selectors/grouping.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "css": {
+    "selectors": {
+      "type": {
+        "__compat": {
+          "description": "Grouping selectors (<code>,</code>)",
```
Comment (ExE-Boss) [inline css/selectors/grouping.json]:
```suggestion
          "description": "Selector list (<code>,</code>)",
```

File: css/selectors/grouping.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          "mdn_url": "https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/Grouping_selectors",
```
Comment (ExE-Boss) [inline css/selectors/grouping.json]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Selector_list",
```

File: css/selectors/grouping.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{
+  "css": {
+    "selectors": {
+      "type": {
```
Comment (ExE-Boss) [inline css/selectors/grouping.json]:
```suggestion
      "list": {
```

Comment (ExE-Boss) [inline css/selectors/grouping.json]:
```suggestion
      "list": {
```

File: css/selectors/grouping.json (hunk 1 above)
Comment (ExE-Boss) [inline css/selectors/grouping.json]:
```suggestion
          "description": "Selector list (<code>,</code>)",
```

File: css/selectors/grouping.json (hunk 2 above)
Comment (ExE-Boss) [inline css/selectors/grouping.json]:
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Selector_list",
```

**Comment by ramiy** [comment]
@ExE-Boss my reference is: https://github.com/mdn/browser-compat-data/pull/532/files

File: css/selectors/grouping.json (hunk 3 above)
Comment (ramiy) [reply css/selectors/grouping.json]: See: https://github.com/mdn/browser-compat-data/pull/532/files

It should be`type` not `list`.

File: css/selectors/grouping.json (hunk 2 above)
Comment (ramiy) [reply css/selectors/grouping.json]: See: https://github.com/mdn/browser-compat-data/pull/532/files

The link should refer to the `Groping selectors` page.

File: css/selectors/grouping.json (hunk 1 above)
Comment (ramiy) [reply css/selectors/grouping.json]: Done.

File: css/selectors/grouping.json (hunk 2 above)
Comment (ramiy) [reply css/selectors/grouping.json]: Done.

File: css/selectors/grouping.json (hunk 3 above)
Comment (ramiy) [reply css/selectors/grouping.json]: Done.

**Comment by vinyldarkscratch** [comment]
Thanks for the quick changes!  One last thing -- may we rename the file to "list.json" to adhere to the rename?

**Comment by ramiy** [comment]
File renamed.

**Pull request merged**